fairscale/optim/oss.py

```diff
@@ -422,69 +465,6 @@ def refresh_trainable(self) -> None:
 
         self._setup_flat_buffers()
 
-    def _broadcast_state_dict(self) -> None:
```

Comment:
consolidate the two following functions into one, since with the "all ranks" behavior the split did not make a lot of sense anymore